app/assets/stylesheets/elements/_list.scss

```diff
@@ -67,3 +68,19 @@
     margin-left: 2rem;
   }
 }
+
+.tools-list {
+  img {
+    height: 5rem;
+
+    @include media($medium-screen) {
+      height: 8rem;
+      max-width: 50%
+    }
+  }
+
+  li {
+    @include padding(0.8rem 2rem);
+    width: 50%;
+  }
+}
```

Comment:
Minor organizational suggestion here… I tend to order nested selectors in roughly the order they appear in markup. So, this would become…

```scss
.tools-list {
  li {
    // styles!
  }

  img {
    // styles!
  }
}
```